Terran-One/cw-simulate-ui
# Added copy to clipboard functionality to Summary tab.

## Issue link

[[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)](https://trello.com/c/1hkxFhnK/101-add-copy-button-to-output-json-renderers)

## Description
Added a generic component for copying to clipboard. This component is right now used for Message section of Summary tab but can pretty much used for any place wherever we want to introduce copy to clipboard functionality.

## Test steps

1. Go to simulation page, perform some executions.
2. Now go to Summary tab, you can see copy icon next to Message label. Click on it and the corresponding message should be copied now.
